internal/services/batch/batch_pool.go

```diff
@@ -1308,9 +1312,15 @@ func flattenBatchPoolNetworkConfiguration(input *pool.NetworkConfiguration) []in
 		dynamicVNetAssignmentScope = string(*input.DynamicVnetAssignmentScope)
 	}
 
+	acceleratedNetworkingEnabled := false
+	if input.EnableAcceleratedNetworking != nil && *input.EnableAcceleratedNetworking {
+		acceleratedNetworkingEnabled = true
+	}
```

Comment:
Since it defaults to false we can use the pointer.From function further down and get rid of this
```suggestion
```

Reply:
Agreed